routes/route.js

```diff
@@ -1,101 +1,152 @@
 const router = require('express').Router();
 
-const  bcrypt = require('bcrypt');
+const bcrypt = require('bcrypt');
 
-const Client = require('../schema/clients')
+const Client = require('../schema/clients');
+const Access = require('../schema/access_token');
```

Comment:
- use index file in schema folder or models folder 
- use destructuring assignment